### PR TITLE
zh-CN: update learn/server-side/express_nodejs/forms/delete_author_form

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

**Comment by PassionPenguin** [comment]
PTAL @yin1999

File: files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ title: 删除作者表单
 slug: Learn/Server-side/Express_Nodejs/forms/Delete_author_form
 ---
 
-此子文档显示，如何定义页面以删除`Author`对象。
+此子文档显示，如何定义页面以删除 `Author` 对象。
 
-正如[表单设计](/zh-CN/docs/Learn/Server-side/Express_Nodejs/forms#form_design)部分所讨论的那样，我们的策略是，只允许删除“未被其他对象引用”的对象（在这种情况下，这意味着如果作者`Author`被一本书`Book`引用，我们将不允许删除作者）。在实现方面，这意味着，表单需要在删除作者之前，先确认没有关联的书籍。如果存在关联的书籍，则应显示它们，并说明在删除`Author`对象之前，必须删除它们。
+正如[表单设计](/zh-CN/docs/Learn/Server-side/Express_Nodejs/forms#form_design)部分所讨论的那样，我们的策略是，只允许删除“未被其他对象引用”的对象（在这种情况下，这意味着如果作者 `Author` 被一本书 `Book` 引用，我们将不允许删除作者）。在实现方面，这意味着，表单需要在删除作者之前，先确认没有关联的书籍。如果存在关联的书籍，则应显示它们，并说明在删除 `Author` 对象之前，必须删除它们。
```
Comment (yin1999) [inline files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md:8]:
```suggestion
正如[表单设计](/zh-CN/docs/Learn/Server-side/Express_Nodejs/forms#表单设计)部分所讨论的那样，我们的策略是，只允许删除“未被其他对象引用”的对象（在这种情况下，这意味着如果作者 `Author` 被一本书 `Book` 引用，我们将不允许删除作者）。在实现方面，这意味着，表单需要在删除作者之前，先确认没有关联的书籍。如果存在关联的书籍，则应显示它们，并说明在删除 `Author` 对象之前，必须删除它们。
```

File: files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md [hunk 2]
```diff
@@ -98,9 +98,9 @@ exports.author_delete_post = function (req, res, next) {
 };
 ```
 
-首先，我们验证是否已提供 id（这是通过表单主体参数发送的，而不是使用 URL 中的版本）。然后我们以与`GET`路由相同的方式，获得作者及其相关书本。如果没有书本，那么我们删除作者对象，并重定向到所有作者的列表。如果还有书本，那么我们只需重新呈现表格，传递作者和要删除的书本列表。
+首先，我们验证是否已提供 id（这是通过表单主体参数发送的，而不是使用 URL 中的版本）。然后我们以与 `GET` 路由相同的方式，获得作者及其相关书本。如果没有书本，那么我们删除作者对象，并重定向到所有作者的列表。如果还有书本，那么我们只需重新呈现表格，传递作者和要删除的书本列表。
 
-> **备注：** 我们可以检查对 `findById()` 的调用，是否返回任何结果，如果没有，则立即呈现所有作者的列表。为简洁起见，我们将代码保留在上面（如果找不到 id，它仍会返回作者列表，但这将在`findByIdAndRemove()`之后发生）。
+> **注意：** 我们可以检查对 `findById()` 的调用，是否返回任何结果，如果没有，则立即呈现所有作者的列表。为简洁起见，我们将代码保留在上面（如果找不到 id，它仍会返回作者列表，但这将在 `findByIdAndRemove()` 之后发生）。
```
Comment (yin1999) [inline files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md:103]: Why changing the prefix "备注：" to "注意：". When you have a look on the preview page, you will find that the note card is not rendered:

image

File: files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md [hunk 3]
```diff
@@ -172,7 +172,7 @@ p
 
 ![](locallibary_express_author_delete_withbooks.png)
 
-> **备注：** 其他删除对象的页面，可以用相同的方式实现。我们把它留下，作为挑战。
+> **注意：** 其他删除对象的页面，可以用相同的方式实现。我们把它留下，作为挑战。
```
Comment (yin1999) [inline files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md:175]: idto

Comment (PassionPenguin) [reply files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md:175]: maybe needs to add a notes for these kinds of translation in the guidelines?

File: files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md (hunk 2 above)
Comment (PassionPenguin) [reply files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md:103]: hmm..should be my fault. got remembered the latter one to be correct.

Comment (PassionPenguin) [reply files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md:103]:
```suggestion
> **备注：** 我们可以检查对 `findById()` 的调用，是否返回任何结果，如果没有，则立即呈现所有作者的列表。为简洁起见，我们将代码保留在上面（如果找不到 id，它仍会返回作者列表，但这将在 `findByIdAndRemove()` 之后发生）。
```

File: files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md (hunk 3 above)
Comment (PassionPenguin) [reply files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md:175]:
```suggestion
> **备注：** 其他删除对象的页面，可以用相同的方式实现。我们把它留下，作为挑战。
```

File: files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md (hunk 2 above)
Comment (yin1999) [reply files/zh-cn/learn/server-side/express_nodejs/forms/delete_author_form/index.md]: You can refer [Markdown in MDN](https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN) to find out all the format we used to document.

**Review by yin1999 — APPROVED** [review]
Thanks @PassionPenguin

**Pull request merged**